### PR TITLE
MEB-26 add link to Flattr profile

### PR DESCRIPTION
**Pull request by caroline-gschwend**

### CODE REVIEW

File: metabrainz/templates/donations/donate.html [hunk 1]
```diff
@@ -127,6 +127,10 @@ <h1 class="page-title">Make a Donation</h1>
       United States
     </p>
     <p>Please make checks payable to the <strong>MetaBrainz Foundation</strong>.</p>
+    
+    <h3>Flattr</h3>
+    <p>Follow this link to pay through Flattr:</p>
+    <a href="https://flattr.com/profile/metabrainz ">https://flattr.com/profile/metabrainz</a>
```
Comment (Freso) [inline metabrainz/templates/donations/donate.html]: This should be encapsulated in `<p>…</p>` too.

File: metabrainz/templates/donations/donate.html [hunk 2]
```diff
@@ -127,6 +127,10 @@ <h1 class="page-title">Make a Donation</h1>
       United States
     </p>
     <p>Please make checks payable to the <strong>MetaBrainz Foundation</strong>.</p>
+    
+    <h3>Flattr</h3>
+    <p>Follow this link to pay through Flattr:</p>
```
Comment (Freso) [inline metabrainz/templates/donations/donate.html]: Perhaps, "If you use Flattr to donate to your favourite projects, you can click below to Flattr MetaBrainz:". @mayhem or Christina may have a suggestion for even better wording.

File: metabrainz/templates/donations/donate.html (hunk 1 above)
Comment (Freso) [inline metabrainz/templates/donations/donate.html]: This should perhaps also be a Flattr button?

**Comment by mayhem** [comment]
:shaved_ice: :sheep: :bus: :nail_care:

**Comment by gentlecat** [comment]
:eyes:

**Pull request merged**